test/http.js

```diff
@@ -1,8 +1,13 @@
+import stream from 'stream';
+import PCancelable from 'p-cancelable';
```

Comment:
No, I just prefer the order:

- builtins
- ava (placed here because it's the main external import)
- external
- local